core/store/src/db.rs

```diff
@@ -85,13 +129,22 @@ pub enum DBCol {
     ColReceiptIdToShardId = 27,
     _ColNextBlockWithNewChunk = 28,
     _ColLastBlockWithNewChunk = 29,
-    /// Map each saved peer on disk with its component id.
+    /// Network related: Map each saved peer on disk with its component id (a.k.a. nonce).
```

Comment:
Is "component" the right word here? This is **not** connected component, right? Maybe just call this `nonce` instead? 

Things like "edges in this component" sound very confusing to me (unless I am already hopelessly confused, and nonces *are* somehow connected components as well).

Reply:
Yes, that is very unfortunate naming - I've updated the comment